themes/wporg-translate-events-2024/blocks/pages/events/event-details/index.php

```diff
@@ -20,9 +21,16 @@ function ( $breadcrumbs ): array {
 				}
 			);
 
+			$event_id = $attributes['event_id'] ?? array();
+			$event = Translation_Events::get_event_repository()->get_event( $event_id );
+			if ( ! $event ) {
+				return '';
+			}
+			$attributes['event'] = $event;
+
 			render_page(
 				__DIR__ . '/render.php',
-				__( 'Event', 'wporg-translate-events-2024' ),
+				__( $event->title(), 'wporg-translate-events-2024' ),
```

Comment:
Never pass variables into translation functions:
```suggestion
				$event->title(),
```

Reply:
Thanks for catching this!